src/sql/platform/connection/common/connectionStore.ts

```diff
@@ -91,6 +91,8 @@ export class ConnectionStore {
 				});
 		} else if (credentialsItem.authenticationType === 'AzureMFA' || credentialsItem.authenticationType === 'dstsAuth' && credentialsItem.azureAccount) {
 			return Promise.resolve({ profile: credentialsItem, savedCred: true });
+		} else if (credentialsItem.authenticationType === 'NoAuth') {
```

Comment:
I would strongly suggest putting comments in all the ADS Core code that NoAuth is currently only supported by the Kusto provider so that people aren't confused about what supports it